spec/std/time/time_spec.cr

```diff
@@ -34,15 +34,15 @@ describe Time do
     time.nanosecond.should eq(999_999_999)
   end
 
-  it "initialize millisecond negative" do
+  it "initialize nanosecond negative" do
```

Comment:
I think this description should be `fail initialize with negative nanosecond` WDYT?

Reply:
Sure. There are many specs in std with very crude labels... ;)